SwedbankPaySDK/Classes/SwedbankPaySDK+Extensions/MerchantBackendConfiguration.swift

```diff
@@ -0,0 +1,207 @@
+            }
+            withTopLevelResources(completion) { topLevelResources in
+                topLevelResources.consumers.post(
+                    api: api,
+                    consumer: consumer,
+                    userData: userData
+                ) {
+                    do {
+                        let viewConsumerIdentification = try $0.get().requireOperation(
+                            rel: Operation.TypeString.viewConsumerIdentification.rawValue
+                        )
+                        let info = ViewConsumerIdentificationInfo(
+                            webViewBaseURL: backendUrl,
+                            viewConsumerIdentification: viewConsumerIdentification
+                        )
+                        completion(.success(info))
+                    } catch let error {
+                        completion(.failure(error))
+                    }
+                }
+            }
+        }
+        
+        public func postPaymentorders(paymentOrder: SwedbankPaySDK.PaymentOrder?, userData: Any?, consumerProfileRef: String?, completion: @escaping (Result<SwedbankPaySDK.ViewPaymentOrderInfo, Error>) -> Void) {
+            guard var paymentOrder = paymentOrder else {
+                fatalError("MerchantBackendConfiguration requires use of PaymentOrder")
+            }
+            if let consumerProfileRef = consumerProfileRef {
+                paymentOrder.payer = .init(consumerProfileRef: consumerProfileRef)
+            }
+            withTopLevelResources(completion) { topLevelResources in
+                topLevelResources.paymentorders.post(
+                    api: api,
+                    paymentOrder: paymentOrder,
+                    userData: userData
+                ) {
+                    do {
+                        let viewPaymentorder = try $0.get().requireOperation(
+                            rel: Operation.TypeString.viewPaymentOrder.rawValue
+                        )
+                        let info = ViewPaymentOrderInfo(
+                            webViewBaseURL: paymentOrder.urls.hostUrls.first ?? backendUrl,
+                            viewPaymentorder: viewPaymentorder,
+                            completeUrl: paymentOrder.urls.completeUrl,
+                            cancelUrl: paymentOrder.urls.cancelUrl,
+                            paymentUrl: paymentOrder.urls.paymentUrl,
+                            termsOfServiceUrl: paymentOrder.urls.termsOfServiceUrl
+                        )
+                        completion(.success(info))
+                    } catch let error {
+                        completion(.failure(error))
+                    }
+                }
+            }
+        }
+        
+        public func decidePolicyForPaymentMenuRedirect(navigationAction: WKNavigationAction, completion: @escaping (SwedbankPaySDK.PaymentMenuRedirectPolicy) -> Void) {
```

Comment:
Codacy found an issue: [Line should be 120 characters or less: currently 168 characters](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-ios/pullRequest?prid=6260585)